daemon/daemon_test.go

```diff
@@ -620,6 +621,62 @@ func TestDaemon_Automated_semver(t *testing.T) {
 	w.ForImageTag(t, d, resid.String(), container, "3")
 }
 
+func TestDaemon_Automated_no_metadata(t *testing.T) {
```

Comment:
A bit tricky verifying that something _doesn't_ happen, isn't it :-/ Is it possible to verify that the update is not processed, by faking `UpdateManifests`, or otherwise getting the result of the `pollForNewImages` calculation? (It occurs to me that factoring the calculation out might be a useful tidy anyway)

Reply:
Not sure if faking `UpdateManifests` and testing if there are no calls made to it makes this test more reliable as you still can't verify why the call hasn't been made (although while I am typing this it occurred to me that combined with the log output it should give you a pretty good idea).

I will look into factoring out the calculation and see if this eases the situation.

Reply:
I factored out the image change calculation. PTAL.